website/docs/language/backend/s3.mdx

```diff
@@ -555,71 +553,28 @@ to only a single state object within an S3 bucket is shown below:
     {
       "Effect": "Allow",
       "Action": ["s3:GetObject", "s3:PutObject"],
-      "Resource": "arn:aws:s3:::example-bucket/myapp/production/tfstate"
-    }
-  ]
-}
-```
-
-The example backend configuration below documents the corresponding `bucket` and `key` arguments:
-
-```hcl
-terraform {
-  backend "s3" {
-    bucket = "example-bucket"
-    key    = "path/to/state"
-    region = "us-east-1"
-  }
-}
-```
-
-Refer to the [AWS documentation on S3 access control](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-access-control.html) for more details.
```

Comment:
Should this link still be preserved after the example configuration that is now below?

Reply:
Good catch, thanks @jar-b! I’ve reverted that line removal in d27513212ed9f836bbb7d40b14e32c28ae372039.